src/Subscriber/Mock.php

```diff
@@ -60,6 +61,20 @@ public function onBefore(BeforeEvent $event)
             }
         }
 
+        $save_to = $event->getRequest()->getConfig()->get('save_to');
```

Comment:
I think scrutinizer could be configured to spot (and create patches for) these kinds of style violations automatically. Anyway fixed.